src/Apps/Order/Routes/Payment/context.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useReducer, createContext, Dispatch } from "react"
+import { paymentReducer } from "./reducer"
+import { PaymentAction, PaymentState } from "./types"
+
+const initialPaymentState = {
+  selectedBankAccountId: "",
+}
+
+const PaymentContext = createContext<{
+  state: PaymentState
+  dispatch: Dispatch<PaymentAction>
+}>({
+  state: initialPaymentState,
+  dispatch: () => null,
+})
+
+const PaymentProvider: React.FC = ({ children }) => {
+  const [state, dispatch] = useReducer(paymentReducer, initialPaymentState)
+
+  // for developers' convenience only
+  if (process.env.NODE_ENV === "development") {
+    console.log({ paymentState: state })
+  }
```

Comment:
I thought it'd be good to log the state to the console, which can help us the way redux-devtools does. the same can also be done for fired actions.

Reply:
We should make the message a bit more verbose via our logger, just so we know it's a dev thing. Something like:
```
import createLogger from 'Utils/logger'

const logger = createLogger("Order/Routes/Payment/PaymentContext.ts") 
...
logger.log(...)